synthesizer/snark/src/certificate/mod.rs

```diff
@@ -37,6 +37,7 @@ impl<N: Network> Certificate<N> {
         proving_key: &ProvingKey<N>,
         verifying_key: &VerifyingKey<N>,
     ) -> Result<Certificate<N>> {
+        #[cfg(feature = "dev-print")]
```

Comment:
Is it possible to enable this feature on WASM? Maybe we should write the following just to be safe
```suggestion
        #[cfg(all(not(feature = "wasm"), feature = "dev-print"))]
```

Reply:
The dev-(e)println macros are already gated by the dev-print feature flag. They dont print anything if its not enabled. And if we did want to add that defense in depth, if it was needed, then we should do it at the declaration of the macros.

Resolving this comment for now!